docs/reference/contrib/modeladmin/menu_item.md

```diff
@@ -46,3 +46,9 @@ Wagtail's 'Explorer' menu item has an order value of `100`, so supply a value gr
 If you'd like the menu item for your model to appear in Wagtail's 'Settings' sub-menu instead of at the top level, add `add_to_settings_menu = True` to your `ModelAdmin` class.
 
 This will only work for individual `ModelAdmin` classes registered with their own `modeladmin_register` call. It won't work for members of a `ModelAdminGroup`.
+
+## `ModelAdmin.exclude_from_menu`
+
+**Expected value**: `True` or `False`
+
+If you'd like this model admin to be excluded from the menu, set to `False`.
```

Comment:
This is the wrong way round - it should be `True` to exclude.

To avoid this sort of issue, I think it's worth choosing a name that avoids double negatives ("don't exclude") - I guess you've chosen it to align with `exclude_from_explorer`, but it doesn't really have much in common with that functionality. It's more related to `add_to_settings_menu` - essentially what you're doing here is turning that from a binary choice (if we're not adding to the settings menu, it MUST be added to the main admin menu instead) into a three-way choice (it can be added to the settings menu, the admin menu, or neither).

I'd suggest naming the option `add_to_admin_menu`, to match the convention we've used for the hook names (`register_admin_menu_item` / `register_settings_menu_item`), and defaulting to True. Then, to keep backwards compatibility, the logic will be:

* if `add_to_settings_menu` is true, add the item to the settings menu and ignore `add_to_admin_menu`
* Otherwise, if `add_to_admin_menu` is true, add the item to the main menu
* Otherwise, don't add it at all